docs/install/dockerhub_builds.md

```diff
@@ -1,30 +1,32 @@
 ## Building Narrative Images with Dockerhub
 (Updated 6/1/2018)
 
-We currently make use of Travis and Dockerhub (https://hub.docker.com) to manage Narrative production builds. There are three images here that stack on each other.
+We currently make use of GitHub Actions and the GitHub Packages (aka the GitHubContainer Repository, https://ghcr.io) to manage Narrative production builds. The narrative comprises two images (to facilitate reasonable narrative build times) and a third tiny image for the narrative version.
```

Comment:
started updating this documentation; will do some more on it when I get this ui-assets thing sorted out